docker-compose-services/php8_1/docker-compose.php8_1.yaml

```diff
@@ -0,0 +1,29 @@
+version: '3.6'
+
+services:
+  php:
+    container_name: ddev-${DDEV_SITENAME}-php
+    image: php:8.1.0alpha1-fpm-buster
```

Comment:
We might want to wait for alpha2 to be released so opcache is available. See https://bugs.php.net/bug.php?id=81133

Reply:
Supposed to be out tomorrow I think. Thanks for all your work on this!